index.html

```diff
@@ -6,9 +6,15 @@
     <meta charset="UTF-8">
     <meta name="description" content="The wedding reception of Alvin and Anita">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script>
+      /* https://stackoverflow.com/questions/3221561/eliminate-flash-of-unstyled-content */
```

Comment:
🤔Should we prevent JS-blockers from viewing the page?
What do you think of Stefan's solution in your link? Would including jQuery be overkill? How bad is this flashing problem right now?

Reply:
Sorry - got a bit lazy with this one and didn't consider all the options. I agree - we shouldn't prevent JS blockers from viewing the page.

Stefan's solution seems solid, but we would be including jQuery just for this purpose which could be overkill. It's not too much code to maintain, however, so I don't think it's all too bad.

Another solution which I thought was better is a CSS solution, suggested in 2 different answers:
* https://stackoverflow.com/a/43823506
* https://stackoverflow.com/a/53364612

The FOUC is not that bad. On the live site it only seems that the image takes some time to load, so there is a white screen instead of the image for a split moment. No content seems to be unstyled. I only noticed the unstyled content when previewing my Github files through https://htmlpreview.github.io/, which is why I decided to address in case it does become a problem.

Reply:
I guess there is one downside to the CSS approach, although I don't think we'll run into this:

"The downside to this approach is if the CSS file fails to load, the page will be blank. For example, naming your CSS file "ads.css" will trigger ad blocker plugins and the file will fail to load, thus leaving the page completely blank. (Of course, that could be intentional...)"

Reply:
Going to drop this for now, unless the FOUC becomes really bad. Here are some notes from my findings.

I was hoping for a solution where the site would wait for the background image to load as well before displaying anything.

The CSS solution doesn't do this, since it displays the elements almost immediately, while the background image is still loading. I implemented the solution involving adding `<style>html{display: none;}</style>` to the `head` and the `html { display: block; }` to the stylesheet linked at the end of the `body`. Moreover, having an additional CSS file to link to contributes to your concern with additional network requests on HTTP 1.1, so this solution is less than ideal.

Using `$(window).on('load', ...)` with jQuery, I am able to wait for the images to load before displaying everything else. However, I'd rather refrain from adding this piece of code for the time being until we actually really need it. I don't really want to add jQuery just for this purpose, and get into using a CDN vs. a local file.
```
 <style type="text/css">
   .hidden {display:none;}
 </style>
 <script type="text/javascript">
   $('html').addClass('hidden');
   $(window).on("load", function() {
     $('html').removeClass('hidden');
   });
</script>
```